### DISCUSSION TITLE
Cannot load images with html-loader in Angular 13.1.x -- works on Angular 13.0.x

### DISCUSSION THREAD

**Discussion by pongells** [webpack/webpack#15465] [Q&A]
Hello, this might not be an issue with `webpack` or the loaders themselves, however I hope to find some help here... I am all out of ideas.

Situation:
- I recently upgraded my app from Angular 12 to Angular 13
- I am using `@angular-builders/custom-webpack` to configure webpack
- nothing changed in my webpack config between the versions (who knows what changed in Angular)
- same version of  `html-loader` in both (latest)

Result:
- Angular 12 loads the images
- Angular 13.1.x does not load the images

Stackblitz: https://stackblitz.com/edit/angular-13-html-loader-images 
(I don't know how to make it work in the web app, but you can download the project do `npm install` and `npm start`)

Related Stackoverflow question https://stackoverflow.com/questions/71264971/cannot-use-images-in-imported-markdown-files-after-updating-to-angular-13 (I discovered it because of markdown, but it seem the issue is in the html-loader, in angular.. or some missing configuration on my part).

Any idea at all about what could be causing this issue? Thank you.

**Comment by vankop** [comment]
please ask in angular related issue tracker..
